src/resources/shared/openshift/setup_azure.md

```diff
@@ -0,0 +1,10 @@
+### Setup Your Azure Profile
+
+Create a service principal and configure its access to Azure resources. Output the result in an Azure SDK compatible auth file and store it
```

Comment:
```suggestion
Create a service principal and configure its access to Azure resources. Output the result in an Azure SDK compatible auth file.
```

I don't think it's necessary to reiterate the throw-away name for the output file in the text description. If we did want this, the current version would also need some grammar tweaks.